database/README.md

```diff
@@ -451,6 +451,24 @@ Indexes:
 - settlements\_tx\_from\_tx\_nonce: btree(`tx_from`, `tx_nonce`)
 - settlements\_tx\_hash: hash(`tx_hash`)
 
+### settlement\_executions
+
+Contains data for each settlement execution of an auction.
+
```

Comment:
There are some solvers which regularly return an error on `/settle` although they bring the tx onchain. I think that fact doesn't have to be encoded in the new table but the readme should probably mention that if you really want to know when an auction actually made it onchain that you'd have to join another table into each row.